### PR TITLE
Update dependencies in xtermjs workspace

### PR DESCRIPTION
**Pull request by jonatanschroeder**
Some dependencies were causing errors on rebuild (see https://github.com/PrairieLearn/PrairieLearn/actions/runs/5801988527/job/15727442552). Tested locally and still works as expected.

### CODE REVIEW

File: workspaces/xtermjs/src/package.json [hunk 1]
```diff
@@ -10,12 +10,12 @@
...
   "dependencies": {
     "command-line-args": "^5.2.1",
-    "command-line-usage": "^6.1.3",
+    "command-line-usage": "^7.0.1",
     "express": "^4.18.2",
     "express-ws": "^5.0.2",
-    "node-gyp": "^9.3.1",
-    "node-pty": "^0.10.1",
-    "xterm": "^5.1.0",
+    "node-gyp": "^9.4.0",
```
Comment (nwalters512) [inline workspaces/xtermjs/src/package.json:16]: Is `node-gyp` still required? I know it was previously used to build `node-pty`, but I'm not sure if any other deps depended on it.

Comment (jonatanschroeder) [reply workspaces/xtermjs/src/package.json:16]: That's a good question, I'll test it later.

Comment (jonatanschroeder) [reply workspaces/xtermjs/src/package.json]: Turns out it's not needed. Removed in 9a703c662.

File: .gitignore [hunk 2]
```diff
@@ -39,6 +39,8 @@ docker-compose.override.yml
 ## Config
 /config.json
 /apps/prairielearn/config.json
+/apps/workspace-host/config.json
+/apps/grader-host/config.json
```
Comment (jonatanschroeder) [inline .gitignore:43]: These are not directly related to the change, but to test the workspace I had to use a config in workspace-host (`"workspacePullImagesFromDockerHub": false`) and noticed it wasn't ignored as it should be.
